subjects/sortedlistmerge/main.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"piscine"
+)
+
+type NodeL struct {
```

Comment:
In the subject and in the rest of the `main` function this structure is called `NodeI` (if I understood well)

Reply:
my mistake, its fixed